### PR TITLE
feat(autorenew): add manual renew action

### PR DESCRIPTION
**Pull request by JeremyDec**
MANAGER-3252

- [ ] : https://github.com/ovh-ux/ovh-ui-angular/pull/427 is required

### CODE REVIEW

File: client/app/billing/autoRenew/autorenew.html [hunk 1]
```diff
@@ -98,6 +98,10 @@
             <extra-top>
                 <oui-action-menu data-text="{{:: 'autorenew_service_action_title' | translate }}">
                     <div data-ng-if="$ctrl.selectedServices.length > 0">
+                        <oui-action-menu-item data-href="{{ $ctrl.manualRenewUrl }}"
```
Comment (FredericEspiau) [inline client/app/billing/autoRenew/autorenew.html]:
```suggestion
                        <oui-action-menu-item data-href="{{:: $ctrl.manualRenewUrl }}"
```

File: client/app/billing/autoRenew/autorenew.html [hunk 2]
```diff
@@ -106,7 +110,7 @@
                             data-on-click="$ctrl.enableBulkAutorenew($ctrl.selectedServices)">
                             <span data-translate="billing_autorenew_service_enable_autorenew"></span>
                         </oui-action-menu-item>
-                        <oui-action-menu-divider ></oui-action-menu-divider>
+                        <oui-action-menu-divider></oui-action-menu-divider>
```
Comment (FredericEspiau) [inline client/app/billing/autoRenew/autorenew.html]: Is it really useful though ? There is no menu item below

File: client/app/billing/autoRenew/autorenew.html (hunk 1 above)
Comment (JeremyDec) [reply client/app/billing/autoRenew/autorenew.html]: No, the `manualRenewUrl` can change, a one time binding won't update the href if you select more services :)

File: client/app/billing/autoRenew/autorenew.html (hunk 2 above)
Comment (JeremyDec) [reply client/app/billing/autoRenew/autorenew.html]: There is more action below ;)
Check https://github.com/ovh-ux/ovh-manager-dedicated/pull/1379/files#diff-87d01eec64e87be9febc1c853f6b41a3R115

**Pull request merged**